File: Java/067. IMC/IMC.java
import java.util.Scanner;
public class IMC
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        double A = scan.nextDouble();
        double P = scan.nextDouble();
        double IMC = P / (A * A);
        if (IMC < 16)
        {
            System.out.println("Magreza grave");
        }
        else if (IMC < 17)
        { 
            System.out.println("Magreza moderada");
        } 
        else if (IMC < 18.5)
        {
            System.out.println("Magreza leve");
        } 
        else if (IMC < 25)
        {
            System.out.println("Saudavel");
        } 
        else if (IMC < 30)
        {
            System.out.println("Sobrepeso");
        } 
        else if (IMC < 35)
        {
            System.out.println("Obesidade Grau I");
        } 
        else if (IMC < 40)
        {
            System.out.println("Obesidade Grau II (severa)");
        } 
        else
        {
            System.out.println("Obesidade Grau III (morbida)");
        } 
        scan.close();
    }
}

File: C++/067. IMC/IMC.cpp
#include <iostream>
int main ()
{
    double A, P;
    double imc;
    std::cin >> A >> P;
    imc = P / (A * A);
    if (imc < 16)
    {
        std::cout << "Magreza grave" << std::endl;
    }
    else if (imc < 17)
    {
        std::cout << "Magreza moderada" << std::endl;
    }
    else if (imc < 18.5)
    {
        std::cout << "Magreza leve" << std::endl;
    }
    else if (imc < 25)
    {
        std::cout << "Saudavel" << std::endl;
    }
    else if (imc < 30)
    {
        std::cout << "Sobrepeso" << std::endl;
    }
    else if (imc < 35)
    {
        std::cout << "Obesidade Grau I" << std::endl;
    }
    else if (imc < 40)
    {
        std::cout << "Obesidade Grau II (severa)" << std::endl;
    }
    else
    {
        std::cout << "Obesidade Grau III (morbida)" << std::endl;
    }
    return 0;
}